contracts/test/mocks/MockTermsContract.sol

```diff
@@ -70,28 +70,36 @@ contract MockTermsContract is MockContract {
     }
 
     function registerTermStart(
-        bytes32 agreementId
+        bytes32 agreementId,
+        address debtor
     )
         public
         returns (bool _registered)
     {
-        functionCalledWithArgs("registerTermStart", agreementId);
+        functionCalledWithArgs("registerTermStart", keccak256(agreementId, debtor));
 
-        return getMockReturnValue("registerTermStart", agreementId) == bytes32(1);
+        return getMockReturnValue(
+            "registerTermStart",
+            keccak256(agreementId, debtor)
+        ) == bytes32(1);
     }
 
-    function mockRegisterTermStartReturnValue(bytes32 agreementId, bool success)
+    function mockRegisterTermStartReturnValue(bytes32 agreementId, address debtor, bool success)
         public
     {
-        mockReturnValue("registerTermStart", agreementId, success ? bytes32(1) : bytes32(0));
+        mockReturnValue(
+            "registerTermStart",
```

Comment:
might be helpful to have a constant for this so we don't mistakenly have a typo.

Reply:
Going to say no on this one, simply because, if we're going to do so, we should do it consistently across all of our mocks -- and that's sufficiently large enough of a task that I don't think it's worth doing (particularly given that these are mocks)